pkg/controllers/kube-apiserver.go

```diff
@@ -113,7 +113,6 @@ func (s *KubeAPIServer) configure(cfg *config.MicroshiftConfig) error {
 	overrides := &kubecontrolplanev1.KubeAPIServerConfig{
 		APIServerArguments: map[string]kubecontrolplanev1.Arguments{
 			"advertise-address": {cfg.NodeIP},
-			"audit-log-path":    {cfg.AuditLogDir},
```

Comment:
It looks like this removes the path setting, but in https://github.com/openshift/microshift/pull/1051#issuecomment-1292300737 you say specifying the path is what turns on audit logging. 

Are we relying on some defaulting behavior elsewhere?

I believe you that this works, I just don't understand how. Sorry to be obtuse.

Reply:
> Are we relying on some defaulting behavior elsewhere?

MicroShift configures its kube-apiserver by merging overrides on top of the base OCP defaults, so this PR is simply removing the override. The value comes from https://github.com/benluddy/microshift/blob/f233b7f9b38eec3c97ad8a2686340b929de9d82e/assets/components/kube-apiserver/defaultconfig.yaml#L45.

Reply:
Aha! Thank you.